src/functionalTest/java/uk/gov/hmcts/reform/wataskmanagementapi/watasks/controllers/PostTaskReplicationMIControllerTest.java

```diff
@@ -161,12 +167,88 @@ public void should_delete_task_and_action_recorded_in_replicate_db() {
             .body("task_history_list.get(1).update_action", equalTo("Configure"))
             .body("task_history_list.get(2).update_action", equalTo("Claim"))
             .body("task_history_list.get(3).state", equalTo("TERMINATED"))
-            .body("task_history_list.get(3).assignee", notNullValue())
+            .body("task_history_list.get(2).assignee", notNullValue())
             .body("task_history_list.get(3).updated_by", notNullValue())
             .body("task_history_list.get(3).updated", notNullValue())
             .body("task_history_list.get(3).update_action", equalTo("AutoCancel"));
 
         common.cleanUpTask(taskId);
     }
 
+    @Test
+    public void user_should_configure_task_and_configure_action_recorded_in_reportable_task() {
+
+        TestVariables taskVariables = common.setupWATaskAndRetrieveIds(
+            "requests/ccd/wa_case_data.json",
+            "processApplication",
+            "process application"
+        );
+        String taskId = taskVariables.getTaskId();
+        common.setupWAOrganisationalRoleAssignment(caseworkerCredentials.getHeaders());
+
+        initiateTask(taskVariables);
+
+        Response result = restApiActions.get(
+            ENDPOINT_BEING_TESTED_TASK,
+            taskId,
+            caseworkerCredentials.getHeaders()
+        );
+
+        result.then().assertThat()
+            .statusCode(HttpStatus.OK.value());
+
+        Response resultReportable = restApiActions.get(
+            ENDPOINT_BEING_TESTED_REPORTABLE,
+            taskId,
+            caseworkerCredentials.getHeaders()
+        );
+
+        resultReportable.prettyPrint();
+        resultReportable.then().assertThat()
+            .statusCode(HttpStatus.OK.value())
+            .body("reportable_task_list.size()", equalTo(1))
+            .body("reportable_task_list.get(0).state", equalTo("UNASSIGNED"))
+            .body("reportable_task_list.get(0).assignee", equalTo(null))
+            .body("reportable_task_list.get(0).updated_by", notNullValue())
+            .body("reportable_task_list.get(0).updated", notNullValue())
+            .body("reportable_task_list.get(0).update_action", equalTo("Configure"));
+
+        common.cleanUpTask(taskId);
+    }
+
+    @Test
+    public void user_should_claim_task_and_claim_action_recorded_in_reportable_task() {
+
+        TestVariables taskVariables = common.setupWATaskAndRetrieveIds("processApplication",
+                                                                       "Process Application");
+        initiateTask(taskVariables);
+
+        common.setupWAOrganisationalRoleAssignment(caseworkerCredentials.getHeaders(), "tribunal-caseworker");
+
```

Comment:
Can we include a check on the reportable here to verify the task has been created? We can then extend the assertions after we claim to confirm that for the given taskId, after claim we see see a difference in state, update action etc from creation?